src/theme/icons.tsx

```diff
@@ -274,6 +274,15 @@ const icons = {
   download: {
     path: <path d="M16 13H13V3H11V13H8L12 17L16 13ZM4 19V21H20V19H4Z" />,
   },
+  expand: {
+    path: (
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```

Comment:
`[change]` We will need to camelCase these or else React will complain

```suggestion
        fillRule="evenodd"
        clipRule="evenodd"
```

Reply:
Updated.